### PR TITLE
New python supervision

### PR DESCRIPTION
**Pull request by Fabien-B**
Complete rewrite of the Python paparazzi center.
Please test it and report any bugs.

### CODE REVIEW

File: sw/supervision/python/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+generated/
```
Comment (gautierhattenberger) [inline sw/supervision/python/.gitignore:1]: maybe it is easier to include the correct generated files (but still keep the .ui files for developers) so standard user can use this without installing development libraries and running the makefile

File: sw/supervision/python/Makefile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+all:
```
Comment (gautierhattenberger) [inline sw/supervision/python/Makefile:2]: add a `clean` target ?

File: sw/supervision/python/app_settings.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+from PyQt5.QtWidgets import *
```
Comment (gautierhattenberger) [inline sw/supervision/python/app_settings.py:1]: add license headers to all python files ?

**Review by gautierhattenberger — CHANGES_REQUESTED** [review]

File: sw/supervision/python/.gitignore (hunk 1 above)
Comment (Fabien-B) [reply sw/supervision/python/.gitignore]: ok, done.

File: sw/supervision/python/Makefile (hunk 2 above)
Comment (Fabien-B) [reply sw/supervision/python/Makefile:3]: done

File: sw/supervision/python/app_settings.py (hunk 3 above)
Comment (Fabien-B) [reply sw/supervision/python/app_settings.py:3]: I added a short 2 line notice. Not a big fan of huge license headers.

**Review by gautierhattenberger — APPROVED** [review]

**Pull request merged**